Assets/Scripts/Screeps_API/ScreepsSocket.cs

```diff
@@ -135,10 +135,24 @@ private void FindSubscription(JSONObject json)
             {
                 return;
             }
-
+            if(list[0].str.IndexOf("room:")==0)
+            {
+                findAdUpdateTimeInRoomSubscription(list[1]);
+            }
             _subscriptions[list[0].str].Invoke(list[1]);
         }
 
+        private void findAdUpdateTimeInRoomSubscription(JSONObject json)
+        {
+            if(!json.HasField("gameTime") || json["gameTime"] == null)
+            {
+                Debug.LogWarningFormat("Can'r find game time in room subscription: {0}", json);
+                return;
+            }
```

Comment:
Hmm, not sure we should log this warning all the time. i'm not sure all subscriptions has gameTime. it seems like `roomMap2` does not contain gameTime 🤔 we might want to be specific about what kind of subscriptions we check for this. e.g. !json.list[0].str.Contains("roomMap2") or something like that 🤷  also the initial subscription on a room as far as I understand does not contain the gametime, but i'm not sure :)

Reply:
I think I forgot to push latest changes